public/dist/app.min.css

```diff
@@ -0,0 +1 @@
ach a.button{margin-top:1rem;font-weight:400;width:100%;padding:1.5rem 1rem;border:1px solid #ffc0cb4d;border-radius:.5rem}.latest-breach a.button:hover{border:1px solid transparent}.latest-breach-info{padding:calc(var(--padding) + 1rem) calc(var(--padding) * 2) calc(var(--padding) * 2) calc(var(--padding) * 2);background-color:rgba(249,249,250,.15);border-top:1px solid #7cdfff0d}.latest-breach .breach-info,.latest-breach .logo-title-wrapper{margin-bottom:1rem}.latest-breach .more-about-this-breach{display:none}@media screen and (max-width:800px){.latest-breach-wrapper{margin:auto 2rem auto}.latest-breach .breach-info,.latest-breach .breach-title.key,.latest-breach a{font-size:.85rem}}@media screen and (max-width:600px){.latest-breach .row{padding:0}.latest-breach-headline{margin-bottom:0;padding:1rem;font-size:.9rem;line-height:150%}.latest-breach .fxm{display:block}}@media screen and (max-width:500px){.latest-breach .logo-title-wrapper{padding:.5rem 0;margin-bottom:0}.latest-breach-wrapper{margin:0;border-radius:0;color:rgba(255,255,255,1);box-shadow:none}.latest-breach-headline{padding:1rem 1rem 1rem 1rem;border-bottom:none;border-top-right-radius:1rem;border-top-left-radius:1rem}.latest-breach-info{border-top:none;padding-bottom:2rem;padding-left:2rem;padding-right:2rem;border-bottom-right-radius:1rem;border-bottom-left-radius:1rem}.latest-breach.container{padding:1rem 0}.latest-breach .breach-info{margin-bottom:.5rem;color:rgba(255,255,255,.6)}}main.container.monitor-homepage{min-height:55vh;padding:0}.monitor-homepage h3{margin-top:var(--margin)}.homepage-scan-form{border:1px dashed var(--borderColor);border-radius:var(--radius)}@media screen and (max-width:800px){.monitor-homepage .row{flex-direction:column}}.scan-results.container{padding-top:0}.found-breaches{display:inline-flex;justify-content:space-between;flex-wrap:wrap;flex-direction:row;padding:0}.show-additional-breaches{flex-wrap:wrap}.scan-results-headline{margin:0}.found-breaches .breach-logo-title.logo-title-wrapper.flx{flex-direction:row-reverse;justify-content:space-between}.found-breaches .breach-listing .logo-wrapper{width:auto;max-width:3rem;max-height:1.25rem;margin:auto 0 auto auto}.found-breaches img.breach-listing-logo{max-width:4rem;max-width:3rem;max-height:1.25rem}@media screen and (max-width:800px){.breach-listing{margin:1.5rem 1rem;flex:1 1 100%;max-width:100%}.found-breaches{padding:0 var(--padding)}}.bullet.light-bullet{border:1px solid rgba(255,255,255,.1)}.small-bullet{height:1rem;width:1rem}.sign-up-banner-button{margin-top:var(--padding);background-color:rgba(255,255,255,1);border:2px solid transparent;color:var(--color)}.sign-up-banner-button:hover{background-color:rgba(255,255,255,.4);box-shadow:none;outline:0;border-color:transparent}.sign-up-feature{margin:.75rem 0;font-weight:300;color:#505054}.sign-up-banner .feature-tip-group{margin-bottom:calc(var(--padding) * 2)}.subpage{margin:auto}.subpage .content-box{flex-direction:column;justify-content:center}
```

Comment:
Not sure if we should check this file in (since it looks generated+minified).

Reply:
Nope, removed it